bazaarvoice/emodb
# retry conditions for Peek operation has been updated

## What Are We Doing Here?

Retry condition for get peek  request of get peek test case has been update to decrease number of false positive test case failures

## How to Test and Verify

Check out this PR
Start emodb on local
Open postman
Import into postman as a collection:
"emodb_tests_databus_get_peek.postman_collection.json"

Import "sor-1-_table.postman_environment.json" as environment variables
Execute tests in postman

Profit
Expected result: All tests are successfully executed on local environment

Risk

Low
There are no changes to application code base

Level

Low

Required Testing
Smoke 

All existing postman tests should be executed without failures.

Risk Summary

There are no risks to application, only tests are involved and it is about splitting existing tests

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
